src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,51 @@
+package duke;
+
+import java.io.IOException;
+
+/**
+ * Main class.
+ *
+ * @author Safwan A0235287X
+ */
+public class Duke {
+
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    /**
+     * Constructor for the Duke main class.
+     * @param filePath Location of the file that stores the data as a result of running the main class.
+     */
+    public Duke (String filePath) {
```

Comment:
Should the space character between `Duke` and `(String filePath)` be removed? According to our coding standard:

> A method or **constructor** name stays attached to the open parenthesis `(` that follows it.